src/SFA.DAS.AssessorService.Api.Types/Models/MergeLogEntry.cs

```diff
@@ -5,11 +5,9 @@ namespace SFA.DAS.AssessorService.Api.Types.Models
     public class MergeLogEntry
     {
         public int Id { get; set; }
-        public string PrimaryEndPointAssessorOrganisationId { get; set; }
         public string PrimaryEndPointAssessorOrganisationName { get; set; }
-        public string SecondaryEndPointAssessorOrganisationId { get; set; }
         public string SecondaryEndPointAssessorOrganisationName { get; set; }
-        public DateTime SecondaryEPAOEffectiveTo { get; set; }
-        public DateTime? CompletedAt { get; set; }
+        public DateTime CompletedAt { get; set; }
```

Comment:
@ChrisJWoodcock is there a reason that `CompletedAt ` is no longer nullable?

Reply:
It never had a null value in it - and I needed the types to match between admin and assessor